src/index.js

```diff
@@ -662,6 +662,15 @@ const members = {
         return ` ${attr.name}${attr.value ? `="${attr.value}"` : ''}`;
       }).join('');
       return `<${name}${attributes}>${this.innerHTML}</${name}>`;
+    },
+
+    set (outerHTML) {
+      if (this.parentNode) {
+        const parsed = parse(outerHTML);
+        this.parentNode.replaceChild(parsed.firstChild, this);
```

Comment:
Have you considered doing `this.parentNode.innerHTML = outerHTML` and let the `innerHTML` do the parsing?